.travis.yml

```diff
@@ -30,12 +30,11 @@ before_install:
 install:
     - sh travis.sh bootstrap
     - sh travis.sh install_package_control
-    - pip3 install python-coveralls pycodestyle;
+    - pip3 install python-coveralls flake8
 
 script:
     - sh travis.sh run_tests --coverage
     - sh travis.sh run_syntax_tests
-    - pycodestyle --statistics --count
```

Comment:
You don't run flake8. 😕

Reply:
Hmm, odd. Dont know why it is removed. Doesnt matter too much, no need to lint on both appveyor and travis. But the better fix it github action https://github.com/divmain/GitSavvy/pull/1170
I do some more work tring to reach him

Reply:
@stoivo you mean to email @divmain to try to enable github actions?

Reply:
I have emailed him and emailed some other of the groups his in to see if they can ask him to concact us. 

I wrote question to you on https://gitter.im/divmain/GitSavvy